src/main/java/seedu/address/ui/UiManager.java

```diff
@@ -20,7 +20,7 @@
     public static final String ALERT_DIALOG_PANE_FIELD_ID = "alertDialogPane";
 
     private static final Logger logger = LogsCenter.getLogger(UiManager.class);
-    private static final String ICON_APPLICATION = "/images/address_book_32.png";
+    private static final String ICON_APPLICATION = "/images/teapot_icon.png";
```

Comment:
Was looking forward to this implementation. Cant wait to see how its gonna look!!